src/app/components/hero/index.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import { Stack, Typography, Container, Divider } from '@mui/material';
+
+interface HeroProps {
+  imageUrl: string;
+  title: string;
+  subtitle?: string;
+  children?: React.ReactNode;
+}
+
+const Hero: React.FC<HeroProps> = ({ imageUrl, title, subtitle, children }) => {
+  return (
+    <Stack
+      sx={{
+        height: '500px',
+        backgroundImage: `url(${imageUrl})`,
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+        position: 'relative',
+        color: 'white',
```

Comment:
Is the color here for the inner text elements? If so I would add color attributes to the specific element and not this parent.

Reply:
This will be fixed in the Theme development by adding the following to each element :
` <Typography variant="subtitle1" sx={{ color: 'text.primary' }}>`

Reply:
I am creating the Theme branch today to begin development.